server/liveblog/embed/embed.py

```diff
@@ -104,6 +105,10 @@ def embed(blog_id, api_host=None, theme=None):
     # retrieve the wanted theme and add it to blog['theme'] if is not the registered one
     try:
         theme_name = request.args.get('theme', theme)
+        theme = get_resource_service('themes').find_one(req=None, name=blog['blog_preferences'].get('theme'))
```

Comment:
o.k., I'll keep it and only do the checkup but I'm trying to say that when we did the checkup 
if theme is None:
            raise SuperdeskApiError.badRequestError(...)
we made sure that if we have "None" value for the theme, it won't pass further. On the other hand, if we do have a theme we continue to use the theme we obtained above using the line no. 108 . Not true?